projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/RunJob.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+
+export interface IRunJobDialogProps {
+    /**
+     * Current Job path
+     */
+    jobPath: string;
+}
+
+
+export default class RunJobDialong extends Component<IRunJobDialogProps> {
```

Comment:
Would it make more sense to implement implements ReactWidget and try avoid using sessionstate?
Also it should be RunJobDialog and not RunJobDialong.

Reply:
Ihaven't noticed the type. I can implement a react widget and use it instead of the showDialog(), but what would changing this class to react widget would bring?

Reply:
In fact leave it the way you have it! 
I was wrong.